R/arg.R

```diff
@@ -137,33 +160,20 @@ arg_match_invalid_msg <- function(arg, values, arg_nm) {
 #' # Succeeds
 #' f(NULL)
 #' @export
-arg_require <- function(arg) {
+arg_require <- function(arg,
+                        error_arg = caller_arg(arg),
```

Comment:
Per your naming scheme, should this be `arg_arg` and `arg_call`?

Reply:
I would prefix with the argument name only when there are two arguments to disambiguate as in `vec_ptype2()`. Normally I'd simply use `arg` and `call`. Otherwise I prefix with `error_` when there are existing arguments with these names as is the case for `arg_require()` and `arg_match()`. I believe this naming scheme should be flexible and intuitive enough for most cases.

Reply:
Ok, makes sense